.github/workflows/ios-build.yml

```diff
@@ -12,10 +12,10 @@ permissions:
 
 jobs:
   build-ios:
-    runs-on: macos-15-xlarge
+    runs-on: macos-15
     env:
       TURBO_CACHE_DIR: .turbo/ios
-      DEVELOPER_DIR: /Applications/Xcode_16.2.app/Contents/Developer
+      DEVELOPER_DIR: /Applications/Xcode_16.4.app/Contents/Developer
```

Comment:
They're rolling out changes to the github actions runners, 16.2 is either not installed, or not pre-loaded with simulators now, so this fixes the CI compile check for ios